README.md

```diff
@@ -1,27 +1,133 @@
 ## Audit scanner
 
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/kubewarden-controller)](https://artifacthub.io/packages/helm/kubewarden/kubewarden-controller)
+[![codecov](https://codecov.io/gh/kubewarden/audit-scanner/graph/badge.svg?token=EDPPGWJFSK)](https://codecov.io/gh/kubewarden/audit-scanner)
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7439/badge)](https://www.bestpractices.dev/projects/7439)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Faudit-scanner.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Faudit-scanner?ref=badge_shield&issueType=license)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Fkubewarden-controller.svg?type=shield&issueType=security)](https://app.fossa.com/projects/custom%2B25850%2Fgithub.com%2Fkubewarden%2Fkubewarden-controller?ref=badge_shield&issueType=security)
 
-> **Note well:** don't forget to checkout [Kubewarden's documentation](https://docs.kubewarden.io)
+> **Note well:** don't forget to checkout [Kubewarden's Audit Scanner documentation](https://docs.kubewarden.io/explanations/audit-scanner)
 > for more information
 
 The Audit scanner inspects the resources defined in the cluster and
 identifies the ones that are violating Kubewarden policies.
 
-The results of the scan can be made available via `PolicyReport` objects. Each Namespace
-has its own dedicated `PolicyReport`. Cluster-wide resources compliance is available via
-the `ClusterPolicyReport` resource.
+The results of the scan are stored in `PolicyReport` and `ClusterPolicyReports` custom resources.
+Each resource has its own dedicated `PolicyReport` or `ClusterPolicyReport`, depending on the type of the resource.
 
-Storing `PolicyReport` and `ClusterPolicyReport` to etcd can be disabled by specifying `--disable-store`.
+See [Querying the reports](#querying-the-reports) for more information.
+
+# Usage
+
+```shell
```

Comment:
Maybe that should be console. I can see keyword highlighting that looks strange when it's shell.